### PR TITLE
Ecs 75 cloudwatch naming conv

### PR DESCRIPTION
**Pull request by irvintim**

### CODE REVIEW

File: asg_drain_source/asg_drain.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+logger.debug("Session is in region %s ", session.region_name)
+
+ec2Client = session.client(service_name='ec2')
+ecsClient = session.client(service_name='ecs')
+asgClient = session.client('autoscaling')
+snsClient = session.client('sns')
+lambdaClient = session.client('lambda')
+
+
+"""Publish SNS message to trigger lambda again.
```
Comment (watchwithmike) [inline asg_drain_source/asg_drain.py]: I've never seen docstrings above the functions they document.  Is this normal?  I've not been writing Python long so I do not know.

File: ecs_capacity_check_source/ecs_capacity_check.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    max_cpu = int(event['CONTAINER_MAX_CPU'])
+    max_mem = int(event['CONTAINER_MAX_MEM'])
+    print('Calculating schedulable containers for cluster %s' % cluster)
+    instance_list = ecs.list_container_instances(cluster=cluster, status='ACTIVE')
+    instances = ecs.describe_container_instances(cluster=cluster,
+                                                 containerInstances=instance_list['containerInstanceArns'])
+ 
+    schedulable_containers = 0
+ 
+    print('\t\tContainers that can be scheduled')
```
Comment (watchwithmike) [inline ecs_capacity_check_source/ecs_capacity_check.py]: Is `print` better than `logger`?  Not sure which output will go where and if we care.

Comment (irvintim) [reply ecs_capacity_check_source/ecs_capacity_check.py]: Lambda takes stdout and puts it in the cloudwatch logs, so it's pretty standard to just use print.

File: asg_drain_source/asg_drain.py (hunk 1 above)
Comment (irvintim) [reply asg_drain_source/asg_drain.py]: I don't know what is standard.  This code was swiped so it was the style of the dude or dudette I grabbed it from.  I can update as you like.

**Comment by watchwithmike** [comment]
I'm closing this as there seems to be a misunderstanding about how we're going to handle lambda jobs.

**Pull request closed without merging**